public/scripts/variables.js

```diff
@@ -901,6 +908,14 @@ export function registerVariableCommands() {
             new SlashCommandNamedArgument(
                 'index', 'list index', [ARGUMENT_TYPE.NUMBER, ARGUMENT_TYPE.STRING], false,
             ),
+            SlashCommandNamedArgument.fromProps({
+                name: 'type',
```

Comment:
Also as discussed on Discord, I think your suggestion of using `cast` instead of `type` is better.
Or even `as`, as that is how many programming languages do casts, and that wording should be clear too.

I feel like the argument name should make sure this is not meant to be as an actual convert/deserialize, but more of a "treat it as this type" kinda cast.
Hard to find the correct wording. But even in the helpstring below, I think calling it "type conversion" is not really correct.